openfe/tests/setup/test_network_planning.py

```diff
@@ -244,6 +244,83 @@ def scorer(mapping):
             scorer=scorer
         )
 
+@pytest.fixture(scope='session')
+def generate_minimal_redundant_network(toluene_vs_others):
+    toluene, others = toluene_vs_others
+    mappers = [BadMapper(), openfe.setup.atom_mapping.LomapAtomMapper()]
+
+    def scorer(mapping):
+        return len(mapping.componentA_to_componentB)
+
+    network = openfe.setup.ligand_network_planning.generate_generate_minimal_redundant_network(
+        ligands=others + [toluene],
+        mappers=mappers,
+        scorer=scorer
+    )
+    return network
+
+def test_generate_minimal_redundant_network(generate_minimal_redundant_network, toluene_vs_others):
+    tol, others = toluene_vs_others
+    assert len(generate_minimal_redundant_network.nodes) == len(others) + 1
+    for edge in generate_minimal_redundant_network.edges:
+        assert edge.componentA_to_componentB != {0: 0}  # lomap should find something
+
+
+def test_generate_minimal_redundant_network_connectedness(minimal_redundant_network):
+    found_pairs = set()
+    for edge in generate_minimal_redundant_network.edges:
+        pair = frozenset([edge.componentA, edge.componentB])
+        assert pair not in found_pairs
+        found_pairs.add(pair)
+
+    assert nx.is_connected(nx.MultiGraph(generate_minimal_redundant_network.graph))
+
+
+def generate_minimal_redundant_network(generate_minimal_redundant_network):
```

Comment:
this method name seems problematic... not a test, so not running as a test, and actually seems to shadow your fixture's name?

Reply:
ah, that's just poor copy-pasting on my part.